src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    private static final int DESCRIPTION_LINE = 0;
+
+    @Override
+    public List<String> readToList(String filePath) {
+        List<String> list;
+        try {
+            list = Files.readAllLines(Path.of(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't find file by path: " + filePath, e);
+        }
+        list.remove(DESCRIPTION_LINE);

```

Comment:
let's better remove the first line in parser, not in reader
because reader should be available for reading any files